README.md

```diff
@@ -86,13 +93,24 @@ You can see the live demo at: https://linkshub.vercel.app
     ```sh
    npm run dev
    ```
-7. Push your changes into your branch
+
+7. Make your changes to the source code.
+
+8. Stage your changes and commit:
+    ```sh
+   git add <filename>
+    ```
+
+    ```sh
+   git commit -m "<your-commit-message>"
+   ```
+9. Push your changes into your branch
    
     ```sh
    git push origin <your_branch_name>
    ```
 
-8. Create a [Pull Request](https://github.com/rupali-codes/LinksHub/compare) 
+10. Create a [Pull Request](https://github.com/rupali-codes/LinksHub/compare) 
    > Click _compare acorss forks_ if you don't see your branch 
 
 #### _Want to add your favorite Links into the Hub? make sure to follow [CONTRIBUTING guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)._
```

Comment:
Lowercase "Links" & "Hubs" (e.g., "links" and "hubs")

Reply:
Sure, I will do it.